test/distributed/test_symmetric_memory.py

```diff
@@ -502,6 +502,29 @@ def test_multimem_one_shot_all_reduce(
         self.assertTrue(res.eq(self.world_size).all().item())
         dist.destroy_process_group()
 
+    @skipIfRocm
```

Comment:
Does this need any additional decorators for e.g., min number of GPUs or P2P support?

Reply:
Good catch. The test requires >=2 GPUs to run.